plugins/sampledata/blog/blog.php

```diff
@@ -119,6 +119,108 @@ public function onAjaxSampledataApplyStep1()
 		$language   = Multilanguage::isEnabled() ? Factory::getLanguage()->getTag() : '*';
 		$langSuffix = ($language !== '*') ? ' (' . $language . ')' : '';
 
+		// Create Field Group for articles
+		$groupTable = new \Joomla\Component\Fields\Administrator\Table\GroupTable($this->db);
+
+		$groupTable->title           = Text::_('PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_FIELDS_GROUP_TITLE') . $langSuffix;
+		$groupTable->description     = '';
+		$groupTable->note            = '';
+		$groupTable->state           = 1;
+		$groupTable->ordering        = 0;
+		$groupTable->created         = Factory::getDate()->toSql();
+		$groupTable->created_by      = $user->id;
+		$groupTable->modified        = Factory::getDate()->toSql();
+		$groupTable->modified_by     = $user->id;
```

Comment:
```suggestion
		$groupTable->modified        = $groupTable->created;
		$groupTable->modified_by     = $groupTable->created_by;
```
When calling getDate 2 times, the times might not be exactly the same. But for something which never has been modified, the modification time has to be exactly the same as the creation time.

For consistency I do the same with the modified_by, even if there will not be a difference.

Reply:
Resolved in the new commit